prompts/large.md

```
@@ -0,0 +1,500 @@
+
+# 6. Generate completion
+parseArger completely my-tool ./my-tool
+source completely.bash
+
+# 7. Test completion
+./my-tool <tab>  # Shows options
+```
+
+## Resources
+
+- GitHub: https://github.com/DimitriGilbert/parseArger
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)